src/test/java/org/opencds/cqf/tooling/utilities/ECQMCreatorTest.java

```diff
@@ -91,32 +92,21 @@ public void TestCMS125FHIR() {
         catch (IOException ioException) {
             ioException.printStackTrace();
         }
+
     }
-   
-    @Test
-    public void TestCMS104FHIR() {
-        // Extract the bundle
-        // NOTE: This is a 2021-AUFHIR measure, this is the test created using TestCMS125FHIR as template
-        ExtractMatBundleOperation o = new ExtractMatBundleOperation();
-        o.execute(new String[] { "-ExtractMATBundle", this.getClass().getResource("ecqm-content-r4-2021/bundles/CMS104-v2-0-004-FHIR-4-0-1.json").getFile() });
 
-        try {
-            Measure measure = refreshMeasure("ecqm-content-r4-2021/input/cql/DischargedonAntithromboticTherapyFHIR.cql", "ecqm-content-r4-2021/input/resources/measure/DischargedonAntithromboticTherapyFHIR.json");
-            assertTrue(null != measure);
-            // Extract data requirements from the measure:
-            List<DataRequirement> drs = new ArrayList<DataRequirement>();
-            for (Extension e : measure.getExtensionsByUrl("http://hl7.org/fhir/us/cqfmeasures/StructureDefinition/cqfm-dataRequirement")) {
-                if (e.hasValue()) {
-                    drs.add(e.getValueDataRequirement());
-                }
-            }
-            assertTrue(!drs.isEmpty());
-            // TODO: Measure-specific validation of data requirements content
-            logger.debug(measureToString(measure));
-        }
-        catch (IOException ioException) {
-            ioException.printStackTrace();
-        }
+    @DataProvider(name="matMeasureLoader")
+    public static Object[][] matMeasures(){
+        return new Object [][] {
+            {"CMS125FHIR-v0-0-004-FHIR-4-0-1.json", "BreastCancerScreeningsFHIR"},
+            {"CMS104-v2-0-004-FHIR-4-0-1.json", "DischargedonAntithromboticTherapyFHIR"},
+        };
+    }
+
+
+    @Test(dataProvider = "matMeasureLoader")
```

Comment:
Using a "DataProvider" like this is the core refactor I'm thinking about for these tests